gdal/swig/python/osgeo/utils/rgb2pct.py

```diff
@@ -125,24 +77,31 @@ def main(argv):
             dst_filename = argv[i]
 
         else:
-            Usage()
+            return Usage()
 
         i = i + 1
 
     if dst_filename is None:
-        Usage()
+        return Usage()
+
+    _ds, err = doit(pct_filename, src_filename, dst_filename, color_count, frmt)
+    return err
+
+
+def doit(pct_filename=None, src_filename=None, dst_filename=None, color_count=256, frmt=None):
+    gdal.AllRegister()
```

Comment:
This won't hurt, but gdal.AllRegister() is called automatically at import time of the bindings. Explicit call to it are legacy

Reply:
OK, I'm removing all calls for gdal.AllRegister() from the utils and samples